pkg/kubelet/pod_workers.go

```diff
@@ -936,7 +942,9 @@ func (p *podWorkers) managePodLoop(podUpdates <-chan podWork) {
 				isTerminal, err = p.syncPodFn(ctx, update.Options.UpdateType, pod, update.Options.MirrorPod, status)
 			}
 
-			lastSyncTime = time.Now()
+			if !utilfeature.DefaultFeatureGate.Enabled(features.EventPLEG) {
+				lastSyncTime = time.Now()
```

Comment:
why does this need to be featureGate-guarded? Is it just trying to save a `time.Now()` call?

Reply:
It's only used when feature gate is disabled. It has no use when feature gate is enabled.

Reply:
Technically the lastSyncTime is "after the last event is retrieved".

Reply:
still thinking we need the source events to be more easily synchronize, (IE. add a counter/timer to status responses, include the status counter/timer and event timer with each container runtime pleg event indicating whether an event occurred after a particular status response.)